compatibility.bs

```diff
@@ -33,9 +33,10 @@ urlPrefix: https://www.w3.org/TR/2011/WD-css3-images-20110217/; spec: css3-image
     text: radial-gradient; url: #ltradial-gradient
     text: repeating-linear-gradient; url: #ltrepeating-linear-gradient
     text: repeating-radial-gradient; url: #ltrepeating-radial-gradient
-urlPrefix: https://drafts.csswg.org/css-backgrounds-3/
+urlPrefix: https://drafts.csswg.org/css-backgrounds-4/
   type: dfn
     text: the backgrounds of special elements;url: #special-backgrounds
+    text: background-clip; url: #background-clip
```

Comment:
I think `#propdef-background-clip` will be the anchor we want.

Reply:
can't double check, because lol

![Screen Shot 2022-06-16 at 10 08 46 AM](https://user-images.githubusercontent.com/67283/174088754-b51d8a27-bf0f-426f-89a5-e3aa79888940.png)

Reply:
it's working *now* ;)
https://drafts.csswg.org/css-backgrounds-4/#background-clip

(it seems a recurrent issues of the CSS specs.)

Reply:
✅

Reply:
heh, broken again :P